academy/ibc-dev/ibcapp_intro.md

```diff
@@ -0,0 +1,32 @@
+# IBC Application Developer Introduction
+
+In the previous sections you've learned how to create custom SDK modules. Additionally, you've had an introduction to IBC, the ibc-go module in the SDK, and how to spin up a relayer to send IBC packets.
```

Comment:
I'm going to include an edit here to make the section work for the IDA and the general portal at the same time without losing the information.